src/cmd/terraform.rs

```diff
@@ -13,22 +13,30 @@ use serde_json::Value;
 
 use crate::cmd::utilities::exec_with_envs_and_output;
 use crate::constants::{KUBECONFIG, TF_PLUGIN_CACHE_DIR};
-use crate::error::SimpleError;
+use crate::error::{SimpleError, SimpleErrorKind};
 
-fn terraform_exec_with_init_validate(
-    root_dir: &str,
-    first_time_init_terraform: bool,
-) -> Result<(), SimpleError> {
+fn terraform_exec_with_init_validate(root_dir: &str) -> Result<(), SimpleError> {
     // terraform init
-    let init_args = if first_time_init_terraform {
-        vec!["init"]
-    } else {
-        vec!["init"]
+    let result = retry::retry(Fibonacci::from_millis(3000).take(5), || {
```

Comment:
please do not use fibonacci but regular retry instead. No need to wait a long time

Reply:
k